justfile

```diff
@@ -176,3 +176,14 @@ load-example-data: devenv
       curl -s https://jobs.opensafely.org/opensafely-internal/tpp-database-schema/outputs/85/download/ --output "$tmp"
       unzip "$tmp" -d "$workspace"
     done
+
+    mkdir -p "$workspace/sub_dir_empty"
+
+    tmp=$(mktemp)
+    # grab published database outputs for example csv and html data
+    curl -s https://jobs.opensafely.org/opensafely-internal/tpp-database-schema/outputs/85/download/ --output "$tmp"
+    unzip -u "$tmp" -d "$workspace"
```

Comment:
I think something's got mangled in the rebase here.  This bit is duplicated in the loop above - it should be in one or the other. Probably we don't need these files and the request duplicated in every workspace, so outside of the loop is enough for sample data

Reply:
arg, yes, thanks